Cohort2/team4/server/.env.sample

```diff
@@ -1 +1,6 @@
-NODE_ENV=development
\ No newline at end of file
+NODE_ENV=development
+PORT
```

Comment:
We shouldn't need all of these properties. I think NodeJS can connect just based on the connection string itself. So we should just need the DATABASE_URL property.